.travis.yml

```diff
@@ -57,27 +85,29 @@ script:
 
   - |
     if [[ "${TYPE}" == "conda" ]]; then
-      python setup.py install;
+      conda install -c mantid/label/nightly mantid-framework
       python setup.py test;
     fi
 
-after_success:
+deploy:
 
-after_success:
-  - |
-    if [[ "${TYPE}" == "conda" ]]; then 
-      test $TRAVIS_BRANCH = "master";
-      conda install conda-build;
-      conda install anaconda-client;
-      bash conda.recipe/conda_upload.sh
-    fi
+ # Deploy pure-python package to PyPI https://pypi.org/project/mantid-total-scattering/
+ - provider: pypi
+   user: mcdonnellmt
+   skip_cleanup: true
+   skip_existing: true
+   on:
+     tags: true
+     branch: master
+   password:
+     secure: kqA6ITb6fr+9Q/KBeU/mxa+d319Uwgc9wi8OPsZssvSneQ8vGAIVSSDBeCVbKSXLjXUIKUWYfndOiqT8wCQU+jpY1TZG0Bm0zWGGxBvcUJEnVrB7siWJ6l+MZ4nPXFFMtwhP0KMDtwen0fLnGOLQlNLyz3qb0Lf8NkM1BumXDK7Lc7p/x8ziKHQothL047jx+0PMsydbTdK2gae/ADm6R4z+ZJirv0/yciUQnCwmiXSnYX5UHtkXqP3k21ZtTV3eA9vNvK1HMEti51lFDrgTt5EBOxtX0UIz6YqfCl5ZHyXMS4uCwqL/m7gRPAc1MTmSurZ4JJkvsaVrU3vUdIimlm2EwqH6+yOfjvWlTdwjglkplt4Z95/rN3CLvYIrYq76XsG2yxY8c11+xmpSMo2/6vbRvmKhmbyV6tFeBgq5rflA/xJkM5WqSyMcE0lhk8xc0RL7aAeblyAs9q/hN9BiZIesegE6pd8DWYEGw6cWnwzaflLgze0nJYE5gQ5z5LZJky8/Nxdxumtf9XVS48PenaNIbDLLG1ER2nufHHg3/5Yv+c0TMRWKv6plb4OLj2AIYa2aSedm7YyVhCbVV0/xM0HxZ8fpKxBMDuFuuU1S1YgQkex7cCfj2QW2JBGoX/iAvPz1sZcU7A+VKaKMJyY+SSmQf+Sz3Px3O8CE+9LerV4=
```

Comment:
This should be set in travis through the web config

Reply:
Done! :+1: